test/fixtures/mson-array-typed-content.json

```diff
@@ -0,0 +1,196 @@
+{
+  "_version": "3.0",
+  "metadata": [],
+  "name": "<API name>",
+  "description": "<API description>\n\n",
+  "element": "category",
+  "resourceGroups": [],
+  "content": [
+    {
+      "element": "category",
+      "content": [
+        {
+          "element": "object",
+          "meta": {
+            "id": "User"
+          },
+          "content": [
+            {
+              "element": "member",
+              "content": {
+                "key": {
+                  "element": "string",
+                  "content": "nontyped array"
+                },
+                "value": {
+                  "element": "array"
+                }
+              }
+            },
+            {
+              "element": "member",
+              "content": {
+                "key": {
+                  "element": "string",
+                  "content": "typed array"
+                },
+                "value": {
+                  "element": "array",
+                  "content": [
+                    {
```

Comment:
There are NO members in this case, just types
` + typed array (array[number,string, boolean])`

If I dont add primitive members w/ null content, nested members info will be lost.

We discuss it yesterday w/ @zdne this is result of our discussion.